packages/core/src/io/node-io.ts

```diff
@@ -34,31 +34,49 @@ import { PlatformIO } from './platform-io';
 export class NodeIO extends PlatformIO {
 	private _fs;
 	private _path;
+	private _fetch;
+	private _httpRegex = /https?:\/\//;
+	public useFetch = false;
 
 	/** Constructs a new NodeIO service. Instances are reusable. */
-	constructor() {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	constructor(fetch?: any) {
```

Comment:
minor: Using `unknown` instead of `any` here should allow you to avoid `eslint-disable-next-line`.

Reply:
Using unknown then causes this._fetch to then be of type unknown, thus causing errors where it is used.

Reply:
This just requires casting to a known type, see [this example](https://github.com/donmccurdy/glTF-Transform/blob/462cd759471d5b65df8d5c54379a09e139785a48/packages/core/src/io/deno-io.ts#L37-L42). I can adjust the TS types after this is merged too though.